test/runtests.jl

```diff
@@ -66,6 +65,7 @@ using SafeTestsets, Test
 
     # Tests extensions.
     @time @safetestset "BifurcationKit Extension" begin include("extensions/bifurcation_kit.jl") end
+    @time @safetestset "Steady State Stability Computations" begin include("miscellaneous_tests/stability_computation.jl") end
```

Comment:
@TorkelE these tests use the HC extension, so they shouldn't be run before the extension section. They were much too early in the testing before I think.

Reply:
It only really used it because I wanted a nice way to compute steady states of systems with multiple systems. I think it make sense to have the extension section only contain tests that actually tests the extensions? If you do not want tests that utilise that code before I could try and come up with another way to compute those steady states.

Reply:
At a minimum if we are using solvers like SteadyStateDiffEq I'd put these after the ODE/SDE/jump tests. So maybe after `    @time @safetestset "Nonlinear and SteadyState System Solving" begin include("simulation_and_solving/solve_nonlinear.jl") end
`?

Reply:
Definitely shouldn't be testing stability before we test solvers though since it uses their results.

Reply:
Also though, if it is relying on the extension's results as a way to validate stuff it should probably come after the tests that validate the extension itself.

Reply:
Just to add -- I like these tests and agree that comparing with HC is a good idea. This is just about ensuring everything the test uses has been tested before this test is run.

Reply:
It is ust impossible to ensure that all tests follow line where everything is introduced only where it is tested (e.g. we are depending on running simulations in most test files, even before we tests that simulations are correct). In principle this might run two superfluous test blocks (if there turns out to be an HC error only caught when that extension is tested), but that seems excessive to move the test lines around?

If you want we can create an additional test block towards the end for "tangential features" of something, but I do think there is something to be gained by grouping related tests together (where admiteldy the common denominator of the misc tests being that they do not really have a common denominator).

Reply:
Yes, I agree that it is impossible in general and we use solving throughout as part of various system tests. But in this case this is testing functionality that is clearly downstream of tests for the ability to accurately solve for steady-states. I moved it to its own section so it is no longer grouped as part of the extension tests.

Reply:
Sounds good for now. 

I think a better argument is that stability (and e.g. serialization) essentially are self-contained features that are less likely to break from changes, and thus should be tested much further back in the pipeline. Meanwhile, the API test maybe should be even earlier than they are.

Generally I do think we should keep the misc category. I think it can be useful for a new users who introduces some feature and tests. In this case they have a default category where they can stick their tests until further notice.